### ISSUE TITLE
Incorrect billing for specific SKUs

### ISSUE THREAD

**Issue by ameukam** [kubernetes/k8s.io#4692]
Since December 2, we are incorrectly billed for `Network Intelligence Center`  service but it's not enabled in the projects of the Kubernetes organization. 

![image](https://user-images.githubusercontent.com/2343515/215220840-20bfb642-ea0a-4cc8-8ff7-783a1c2c8bfc.png)

Apparently this is not new : https://www.googlecloudcommunity.com/gc/Cloud-Hub/How-to-stop-Network-Intelligence-Center-related-SKUs/m-p/502521

/milestone v1.27
/area infra
/priority critical-urgent

**Comment by ameukam** [comment]
cc @jimangel

**Comment by jimangel** [comment]
@ameukam and I synced up last week and discovered that there is a credit for 100% of the billing applied at the end of each month.

This has also been reconfirmed by Manasa (manasach@) on [the linked GCP forum](https://www.googlecloudcommunity.com/gc/Cloud-Hub/How-to-stop-Network-Intelligence-Center-related-SKUs/m-p/515742/highlight/true#M1782).

As @ameukam and I discussed, we think there is a way to improve the query(s) to exclude the credited SKU data or plan for it.
